SignInCheckIn/MinistryPlatform.Translation/Repositories/ChildSigninRepository.cs

```diff
@@ -175,13 +175,15 @@ public List<MpEventParticipantDto> CreateEventParticipants(List<MpEventParticipa
             {
                 "Event_Participant_ID",
                 "Event_ID",
-                "Participant_ID",
+                "Participant_ID_Table_Contact_ID_Table.[First_Name]",
+                "Participant_ID_Table_Contact_ID_Table.[Last_Name]",
+                "Participant_ID_Table.[Participant_ID]",
                 "Participation_Status_ID",
                 "Time_In",
                 "Time_Confirmed",
                 "Time_Out",
-                "Notes",
-                "Domain_ID",
+                // "Participant_ID_Table.[Notes]",
+                // "Domain_ID",
                 "Group_Participant_ID",
```

Comment:
My guess is you are calling off of the participant table and therefore don't need to reference it and just call Notes.  Now Domain_ID you might need to reach the table.  If these are not needed though don't put them in.